tests/operator_test.go

```diff
@@ -223,11 +223,10 @@ var _ = Describe("[Serial][sig-operator]Operator", func() {
 		}
 
 		sanityCheckDeploymentsDeleted = func() {
-
 			Eventually(func() error {
 				for _, deployment := range []string{"virt-api", "virt-controller"} {
 					_, err := virtClient.AppsV1().Deployments(flags.KubeVirtInstallNamespace).Get(context.Background(), deployment, metav1.GetOptions{})
-					if err != nil && !errors.IsNotFound(err) {
+					if err == nil || !errors.IsNotFound(err) {
```

Comment:
Sorry for not noticing this earlier, but in this way you will not check all the deployments. For example, if `virt-api`, in the first iteration, is returned because it is still not deleted the function returns nil and the `Eventually` exits with success and this is wrong. I think that if we want to check that deployments are deleted we should do something like:
```
Eventually(func() bool {
	deploymentCount := 2
	for _, deployment := range []string{"virt-api", "virt-controller"} {
		_, err := virtClient.AppsV1().Deployments(flags.KubeVirtInstallNamespace).Get(context.Background(), deployment, metav1.GetOptions{})
		if err != nil && errors.IsNotFound(err) {
			deploymentCount--
		}
	}
	return deploymentCount == 0
}, 15*time.Second, 1*time.Second).Should(BeTrue())
```

Reply:
Good catch, you're absolutely right... The other fix would be to return an error when `err == nil`, but it's more confusing than your solution.
Will fix!

Reply:
Done.